cli/cmd/version.go

```diff
@@ -121,10 +121,10 @@ func versionCheck() (*lwupdater.Version, error) {
 	}
 
 	if sdk.Outdated {
-		cli.OutputHuman(fmt.Sprintf(
+		fmt.Fprintf(os.Stderr,
```

Comment:
Could you move this to `cli/cmd/outputs.go` new function `cli.OutputHumanErr` or the like.

Reply:
This ☝🏽

Reply:
Added